src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -274,3 +279,19 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='Access key for the given IoT Plug and Play repository.')
         c.argument('user_role', options_list=['--role'], arg_type=get_enum_type(UserRole),
                    help='User role of the access key for the given IoT Plug and Play repository.')
+
+    with self.argument_context('iot central app') as c:
+        c.argument('app_name', app_name_type, options_list=['--name', '-n'])
+
+    with self.argument_context('iot central app create') as c:
+        c.argument('app_name', completer=None)
+        c.argument('location', get_location_type(self.cli_ctx),
+                   help='Location of your IoT Central application. Default is the location of target resource group.')
+        c.argument('sku', arg_type=get_enum_type(AppSku),
+                   help='Pricing tier for IoT Central applications. Default value is ST2.')
+        c.argument('subdomain',
+                   help='Subdomain for the IoT Central URL. Each application must have a unique subdomain.')
+        c.argument('template',
```

Comment:
Are there allowed values for template?

Reply:
Yes, there is. We have an API that allows customers to see the available options.

Reply:
We do not want to hardcode these values in the Azure CLI

Reply:
In this way can we put the API in help message? Because for me (a new user), I totally don't know which value is available for template.

Reply:
Yeah, I can see how that can be confusing. We have an item in our backlog to rewrite the documentation for Swagger and Az CLI. I added some details about the issue of not seeing available templates so we can address it when we have the capacity.